pkg/blackboxexporter/blackboxexporter.go

```diff
@@ -0,0 +1,243 @@
+package blackboxexporter
+
+import (
+	"context"
+
+	"github.com/go-logr/logr"
+	"github.com/openshift/route-monitor-operator/api/v1alpha1"
+	"github.com/openshift/route-monitor-operator/pkg/const/blackbox"
+	"github.com/openshift/route-monitor-operator/pkg/util/reconcile"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	//api's used
+
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	//local packages
+)
+
+type BlackboxExporter struct {
+	Client client.Client
+	Log    logr.Logger
+	Ctx    context.Context
+}
+
+func NewBlackboxExporter(client client.Client, log logr.Logger, ctx context.Context) *BlackboxExporter {
```

Comment:
this doesn't fit what I did in the other packages.. do you want to fix the others or fix this one?

what I had was `func New(...)` which would make using it look like:

```
b := blackboxexporter.New(...)
```

Reply:
sure